src/org/labkey/test/components/pipeline/PipelineTriggerWizard.java

```diff
@@ -183,11 +189,17 @@ public PipelineTriggerWizard setParameterFunction(String function)
         return this;
     }
 
-    public PipelineTriggerWizard addCustomParamter(String key, String value, @NotNull Integer index)
+    public PipelineTriggerWizard showAdvanced()
+    {
+        elementCache().showAdvanced.click();
+        return this;
```

Comment:
This sort of method usually does better if there's a related check to see if the desired state is already present, like 'isAdvancedShown()'
so, if (!isAdvancedShown())
     elementCache.showAdvanced.click()
     waitFor(()-> isAdvancedShown, "didn't happen in time", 2000)
might be how showAdvanced() might work